src/model/enums/piece-type.ts

```diff
@@ -5,6 +5,7 @@ export enum PieceType {
   GRAPHIC = 'GRAPHIC',
   SERVER = 'SERVER',
   AUDIO = 'AUDIO',
-  TRANSITION = 'TRANSITION'
+  TRANSITION = 'TRANSITION',
+  VIDEO_SWITCHER = 'VIDEO_SWITCHER'
```

Comment:
Would be nice if we could call it `VIDEO_MIXER` instead of switcher. Switcher is the American term.

Reply:
Have renamed all `Video Switcher` to `Video Mixer`.